elements/pfe-tabs/src/pfe-tab.scss

```
@@ -3,17 +3,21 @@
 $LOCAL: tabs;
 
 $LOCAL-VARIABLES: (
-  BackgroundColor: transparent,
-  Color:           pfe-var(ui-disabled--text),
-  link:            pfe-var(link),
-  focus:           pfe-var(link--focus),
-  highlight:       pfe-var(ui-accent),
+  BackgroundColor:       transparent,
+  Color:                 pfe-var(text--muted),
+  link:                  pfe-var(link),
+  text-hover:            #151515,
+  focus:                 pfe-var(link--focus),
+  highlight:             pfe-var(ui-accent),
+  BorderWidth--hover:    #{pfe-var(surface--border-width--active)},
+  BorderColor--hover:    pfe-var(surface--base),
+  darkBackgroundColor:   #3c3f42,
   tab: (
-    FontSize:       pfe-var(font-size),
-    PaddingTop:     calc(#{pfe-var(container-padding)} * .666),
-    PaddingRight:   calc(#{pfe-var(container-padding)} * 1.5),
-    PaddingBottom:  calc(#{pfe-var(container-padding)} * .666),
-    PaddingLeft:    calc(#{pfe-var(container-padding)} * 1.5),
+    FontSize:            pfe-var(font-size),
+    PaddingTop:          calc(#{pfe-var(container-padding)}),
```

```suggestion
    PaddingTop:          pfe-var(container-padding),
```
No calc necessary here